conf/evolutions/default/221.sql

```diff
@@ -0,0 +1,12 @@
+# --- !Ups
+CREATE TABLE pano_history (
+   pano_id TEXT NOT NULL,
+   visited_timestamp TIMESTAMP,
+   pano_date TEXT NOT NULL,
```

Comment:
Sorry to keep having you rename things! Let's call this `capture_date` bc that's the name we use in the `gsv_data` table.